src/components/DashboardComponents/AverageChart.js

```diff
@@ -18,7 +18,10 @@ const AverageChart = () => {
 
   return(
     <div className = "text-component">
-      <div className = "text-component__top-right">{minute} m {seconds} sek</div>
+      {minute == 0 ? 
```

Comment:
Good conditional, makes it prettier. Perhaps there should be default values when the equation returns NaN too?